tests/test_A1_csv_to_json.py

```diff
@@ -4,8 +4,7 @@
 
 import src.A1_csv_to_json as load_data_from_csv
 
-elements = os.path.join(REPO_PATH, "tests", "inputs", "csv")
-
+elements = os.path.join(REPO_PATH, "tests", "inputs", "csv_elements")
```

Comment:
As per #135 all csv input files should be in the folder `csv_elements`. Probably, the name of the folder simply has to be changed.

Reply:
I think this should be stored in a variable `CSV_ELEMENTS`. I would store it in a module under `src/constants.py`